platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py

```diff
@@ -499,16 +499,20 @@ def get_change_event_for_module_host_management_mode(self, timeout):
                     s.on_event(event)
                     
                 if s.in_stable_state():
+                    if s.get_control_type() == 0 and not self.sfp_module.SFP.wait_sfp_eeprom_ready([s], 2):
```

Comment:
no need check s.get_control_type() == 0 here, it will be checked inside wait_sfp_eeprom_ready

Reply:
removed redundancy check